vpt/cli/check.py

```diff
@@ -14,33 +14,29 @@ def check():
     print('Checking the devices for 5s...')
 
     # Create capture nodes
+    audio_device = sd.query_devices(kind='input', device=args['audio'])
```

Comment:
I think the `kind` argument is ignored when `device` is passed

Reply:
Well, whatever) it worked alright when I tried it